### PR TITLE
update realm file with default role and redirect URIs

### PR DESCRIPTION
**Pull request by awolfe76**
This updates the realm to include the correct `redirectUris` and `defaultRoles`.
Edit: It now also prevents access to the account management screens.

These diffs are hard to view because the export isn't consistent ... see inline comments.

### CODE REVIEW

File: kubernetes/keycloak/realm.json [hunk 1]
```diff
@@ -34,29 +34,572 @@
...
+  "clients": [
+    {
+      "id": "c749d73d-de71-42fa-aff3-44456890e908",
+      "clientId": "hmda2-api",
+      "surrogateAuthRequired": false,
+      "enabled": true,
+      "clientAuthenticatorType": "client-secret",
+      "secret": "**********",
+      "defaultRoles": ["hmda-user"],
+      "redirectUris": [
```
Comment (awolfe76) [inline kubernetes/keycloak/realm.json]: Most of these are to allow for local development, the most important one for our cluster environments is the relative path, `/*`.

File: kubernetes/keycloak/realm.json [hunk 2]
```diff
@@ -34,29 +34,570 @@
...
+  ],
+  "clients": [
+    {
+      "id": "c749d73d-de71-42fa-aff3-44456890e908",
+      "clientId": "hmda2-api",
+      "surrogateAuthRequired": false,
+      "enabled": true,
+      "clientAuthenticatorType": "client-secret",
+      "secret": "**********",
+      "defaultRoles": ["hmda-user"],
```
Comment (awolfe76) [inline kubernetes/keycloak/realm.json]: This sets the `hmda-user` role as the default for the `hmda2-api` client.

File: kubernetes/keycloak/realm.json [hunk 3]
```diff
@@ -34,29 +34,570 @@
...
+    {
+      "id": "63989b58-df41-4462-b339-055e842c1bb0",
+      "clientId": "account",
+      "name": "${client_account}",
+      "baseUrl": "/auth/realms/hmda2/account",
+      "surrogateAuthRequired": false,
+      "enabled": true,
+      "clientAuthenticatorType": "client-secret",
+      "secret": "**********",
+      "defaultRoles": [],
```
Comment (awolfe76) [inline kubernetes/keycloak/realm.json]: Removing the default roles in the `account` client prevents users from accessing the account mgmt screens.

**Comment by awolfe76** [comment]
@omnipresent07 @kgudel - This should be ready for review.

**Comment by omnipresent07** [comment]
This LGTM 👍

**Pull request merged**